src/Http/Requests/UpdateNavigationRequest.php

```
@@ -0,0 +1,87 @@
+		];
+	}
+
+	/**
+	 * Rejects a bare-list `content` payload — see the mirrored method on
+	 * {@see StoreNavigationRequest::envelopeShapeRule()} for why.
+	 *
+	 * @since 1.0.0
+	 */
+	protected function envelopeShapeRule(): Closure
+	{
+		return function ( string $attribute, mixed $value, Closure $fail ): void {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$attribute'. (undefined)

(UnusedFormalParameter)